trackintel/geogr/distances.py

```diff
@@ -161,4 +162,17 @@ def meters_to_decimal_degrees(meters, latitude):
     float
         An approximation of a distance (given in meters) in degrees.
     """
-    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
\ No newline at end of file
+    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
+
+
+def check_crs(data):
+    wgs = False
+    if data.crs == 4326:
+        wgs=True
+        warnings.warn('Your data is in WGS84, for length calculation the haversine distance is used')
+    elif data.crs == None:
+        wgs=True
+        warnings.warn('Your data is not projected. WGS84 is assumed and for length calculation the haversine distance is used')
+    elif data.crs.is_geographic:
+        raise UserWarning('Your data is in a geographic coordinate system, length calculation fails')
```

Comment:
I think a "check" function should not raise exceptions. It should rather return the result of the check and leave the raising of warnings or exceptions to the specific application (the need for exceptions might be different for different applications).